core/rawdb/accessors_chain.go

```diff
@@ -1196,11 +1196,11 @@ func ReadEtxSet(db ethdb.Reader, hash common.Hash, number uint64) types.EtxSet {
 		return nil
 	}
 	var entries []etxSetEntry
-	if err := rlp.Decode(bytes.NewReader(data), entries); err != nil {
+	if err := rlp.Decode(bytes.NewReader(data), &entries); err != nil {
```

Comment:
this line is modified in one of the previous commits as well. 7b8e9de25cd42 commit

Reply:
Good catch

Reply:
I've squashed and force pushed